doc/docs.md

```diff
@@ -604,33 +591,73 @@ numbers := {
 }
 ```
 
-## Module imports
+## Modules
+
+Simple programs don't need to have a module name - it defaults to 'main'.
+Every file in the root of a folder is part of the same module.
+For information about creating a named module, see [Modules 2](#modules-2).
+
+### Symbol visibility
+
+```v
+pub fn public_function() {
+}
+
+fn private_function() {
+}
+```
+
+Functions are private (not exported) by default.
+To allow other modules to use them, prepend `pub`. The same applies
+to constants and types. 
 
-For information about creating a module, see [Modules](#modules)
+Note: `pub` can only be used from a named module.
 
-### Importing a module
```

Comment:
I think the `Module imports` anchor link would be used instead of this, but if you like I can manually add an anchor for this.